### PR TITLE
Token documentation

### PR DESCRIPTION
**Pull request by george-edgedelta**
Hi @LongTuanNguyen 

Please review the newly created documentation for tokens for technical accuracy.

### CODE REVIEW

File: docs/appendices/tokens.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+|----------------------|---------------------------------------------------------------------------------------------------------------------|
+| Accesses             | This permission gives you read / write access to manage the users in your account.                                  |
+| Rehydrations         | This permission gives you read / write access to manage rehydrations.                                               |
+| Integrations         | This permission gives you read / write access to manage integrations with third-party services.                     |
+| Monitors             | This permission gives you read / write access to manage monitors and custom alert definitions.                      |
+| Agent Configurations | This permission gives you read / write access to manage the configurations for the agents that run locally.         |
+| Hosted Agents        | This permission gives you read / write access to manage the agents hosted by Edge Delta.                            |
+| Tokens               | This permission gives you read / write access to manage tokens.                                                     |
```
[the remark this answers is not in the document]
Comment (george-edgedelta) [reply docs/appendices/tokens.md:31]: Hi @taylanisikdemir and @tjb1019, 

The portal still shows token. Is this actively being worked on? If not, I'll go ahead and publish this document while we wait for these changes to be worked on.

File: docs/appendices/tokens.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+## Create a Token
+
+After you create the token, the unique token key will only display once. Afterwards, you will not be able to retrieve the token. As a result, copy and store the key in a secure location.   
+
+1. In the Edge Delta Admin portal, on the left-side navigation, click **Global Settings**. 
+2. In the Tokens sections, click  **Create Token**. 
+3. Under **Name**, enter a descriptive name for the token, such as **john-smith-integrations-token**. 
+4. Click **Add Permissions**.
+5. Under **Select An Access**, click the drop-down menu, and then select the desired access type.
```
[the remark this answers is not in the document]
Comment (george-edgedelta) [reply docs/appendices/tokens.md:48]: Hi @taylanisikdemir and @tjb1019, 

The portal still shows "select an access." Is this actively being worked on? If not, I'll go ahead and publish this document while we wait for these changes to be worked on.

[the remark this answers is not in the document]
Comment (taylanisikdemir) [reply docs/appendices/tokens.md:48]: @tjb1019 is that something we can address soon so documentation matches prod?

[the remark this answers is not in the document]
Comment (tjb1019) [reply docs/appendices/tokens.md:48]: making these changes to the UI now.  i'll have a PR up shortly.

[the remark this answers is not in the document]
Comment (tjb1019) [reply docs/appendices/tokens.md:48]: https://github.com/edgedelta/edgedelta/pull/3095

[the remark this answers is not in the document]
Comment (taylanisikdemir) [reply docs/appendices/tokens.md:48]: thanks!

**Pull request merged**